PubSub/tests/Unit/MessageTest.php

```diff
@@ -87,6 +88,11 @@ public function testAckId()
         $this->assertEquals(1234, $this->message->ackId());
     }
 
+    public function testDeliveryAttempt()
```

Comment:
Should we add a unit test for null delivery attempt?

Reply:
sure thing! done.